index.html

```diff
@@ -4,18 +4,158 @@
 </head>
 
 <body>
+    <header>
+        <div class="name-title">
+          <div class="blueSquer-p">
+            <div class="blueSquer"></div> 
+            <h3 lass="firstLastName">Maya Nelson</h3>
+          </div>  
+          <p class="title">PROJECT MANAGER</p>     
+                      
+                
+        </div>
+        
+        
+        <nav role="navigation">
+            <div id="menuToggle">
+              <!--
+              A fake / hidden checkbox is used as click reciever,
+              so you can use the :checked selector on it.
+              -->
+              <input type="checkbox" />
+              
+              <!--
+              Some spans to act as a hamburger.
+              
+              They are acting like a real hamburger,
+              not that McDonalds stuff.
+              -->
+             <div class="span-1"> <span></span></div>
+             <div class="span-2"> <span></span></div>
+             <div class="span-3"> <span></span></div>
+              
+              <!--
+              Too bad the menu has to be inside of the button
+              but hey, it's pure CSS magic.
+              -->
+              <ul id="menu">
+                <a href="#"><li>ABOUT ME</li></a>
+                <a href="#"><li>RESUME</li></a>
+                <a href="#"><li>PROJECTS</li></a>
+                <a href="#"><li>CONTACT</li></a>
+                
+              </ul>
+            </div>
+          </nav>
+        
+          
+        
+    </header>
+    <main>
+        <section class="blanck"></section>
+        <div class="introduction">
+        <section class="professional-introduction">
+            <img class="personal-image" src="img/personal-image.jpeg" alt="personal image">
+            <div class="name-family">
+                <h1>Maya</h1>
+                <h1>Nelson</h1>
+            </div>  
+            <hr class="divider"/>
+
+            <div class="buttun-professional">
+              <button class="resume"><b>RESUME</b></button>
+              <button class="project"><b>PROJECT</b></button>
+            </div>
+
+            <div class="follow-links-container">
+            <div class="follow-links-1">
+                <a href="https://en-gb.facebook.com/" class="links-socialMedia"><img
+                   src="/img/Facebook.png" class="images_socialMedia "
+                   alt="facebookImage" /></a>
+                <a href="https://twitter.com/?lang=en" class="links-socialMedia" ><img
+                   src="/img/twitter.png" class="images_socialMedia " alt="TwitterLogo" /></a>
+                <a href="https://uk.linkedin.com/" class="links-socialMedia"><img
+                   src="/img/free-linkedin-icon-112-thumb.png" class="images_socialMedia" alt="LinkedLogo" /></a>
+                <a href="https://www.instagram.com/" class="links-socialMedia"><img
+                   src="/img/Instagram.png" class="images_socialMedia " alt="Instagram" /></a>
```

Comment:
Looks great, just a heads up - `alt` attribute specifies an alternate text for an image, if the image cannot be displayed. You'd typically want use spaces. 
```suggestion
                <a href="https://en-gb.facebook.com/" class="links-socialMedia"><img
                   src="/img/Facebook.png" class="images_socialMedia "
                   alt="Facebook icon" /></a>
                <a href="https://twitter.com/?lang=en" class="links-socialMedia" ><img
                   src="/img/twitter.png" class="images_socialMedia " alt="Twitter icon" /></a>
                <a href="https://uk.linkedin.com/" class="links-socialMedia"><img
                   src="/img/free-linkedin-icon-112-thumb.png" class="images_socialMedia" alt="LinkedLogo" /></a>
                <a href="https://www.instagram.com/" class="links-socialMedia"><img
                   src="/img/Instagram.png" class="images_socialMedia " alt="Instagram icon" /></a>
```

Reply:
:+1: updated ,thank you